src/olympia/users/management/commands/createsuperuser.py

```diff
@@ -80,6 +90,23 @@ def handle(self, *args, **options):
                 'api-secret': apikey.secret
             }))
 
+        if options.get('ui_test', False):
+            # json object for variables file
+            variables = {}
+            variables['api'] = {}
+
+            # export to variables.json object
+            variables['api'].update({
+                'olympia.dev': dict([
+                    ('username', user.username,),
+                    ('jwt_issuer', apikey.key),
+                    ('jwt_secret', apikey.secret)
+                ])
+            })
+
+            with open('tests/ui/variables.json', 'w') as outfile:
+                json.dump(variables, outfile, indent=2)
+
```

Comment:
the command would be `--save-api-credentials=path/to/variables.json` with that then I think